docs/SPEC.md

```diff
@@ -1354,6 +1354,12 @@ So, you have to:
 
 Flux provides a standard library of functions. Find documentation here https://docs.influxdata.com/flux/latest/stdlib/
 
+#### Experimental namespace
+
+Within the standard library there is an `experimental` package namespace.
```

Comment:
Be more specific that anything _under_ this package namespace is included too such as `experimental/date`. Also, be sure to mark that breaking changes may occur within the same edition and we reserve the right to remove values from older editions.

Basically be as explicit as possible for all the ways that we could not support something in experimental. Some might assume that, "Well, it's in experimental, but it's still there and this is an old edition and it's now in the formal standard library in a new edition so it's guaranteed not to be removed."

I want to dispel the "it's guaranteed to not be removed" part of that sentence. Everything else is logical and makes sense. And, in practice, nothing they say is a big problem and is 99% likely to be correct.